uvo_scraper.rb

```diff
@@ -0,0 +1,29 @@
+require "curb"
+
+class UvoScraper
+  SEARCH_URL = "https://www2.uvo.gov.sk/evestnik?p_p_id=evestnik_WAR_eVestnikPortlets&p_p_lifecycle=1&p_p_state=normal&p_p_mode=view&p_p_col_id=column-2&p_p_col_pos=1&p_p_col_count=2".freeze
+  NEW_ISSUE_URL = "https://www2.uvo.gov.sk/evestnik/-/vestnik/aktual".freeze
+  IT_CONTRACTS_CODE = 72
+
+  def initialize(parser_class, release_date, html_client = Curl)
+    @parser = parser_class
+    @html_client = html_client
+    @release_date = release_date
+  end
+
+  def issue_ready?
+    html = @html_client.get(NEW_ISSUE_URL).body
+    header = @parser.new(html).parse_issue_header
```

Comment:
Toto trosku smrdi, nevidim velky zmysel vytvarat novu instanciu pre kazde html. Posunme vklude do `parse_issue_header(html)`. Inak `Parser` nemusi byt ani instancia, stacili by tam staticke metody. Nema to ziadny stav.